.github/workflows/qctools.yml

```diff
@@ -300,15 +293,17 @@ jobs:
     - name: Install FFmpeg
       run: |
         cd ..
-        $FFMPEG_VERSION = "ffmpeg-4.2.1-win64"
+        $FFMPEG_LATEST_WIN64 = "ffmpeg-4.2.1-win64"
```

Comment:
My approval was too quick.
I was willing to say to change to `ffmpeg-latest-win64` for the right part, i.e. really using the latest version available (including unreleased stuff, so the Git master as previously).

Reply:
Changed that, sorry for the misunderstanding :)